kong/plugins/rate-limiting/handler.lua

```diff
@@ -124,12 +125,11 @@ function RateLimitingHandler:access(conf)
 
   local usage, stop, err = get_usage(conf, identifier, current_timestamp, limits)
   if err then
-    if fault_tolerant then
-      kong.log.err("failed to get usage: ", tostring(err))
-    else
-      kong.log.err(err)
-      return kong.response.exit(500, { message = "An unexpected error occurred" })
+    if not fault_tolerant then
```

Comment:
Why `if not` instead of `if` ?

Reply:
It follows out style of `return early`, aka makes it return early. But I can change it back.